### DISCUSSION TITLE
OAuth and OAuth2 support for `http remote`

### DISCUSSION THREAD

**Discussion by raman-r-4978** [treeverse/dvc#5208] [New Features & Ideas]
Hi,

I have a quick question w.r.t https://github.com/iterative/dvc/pull/3343, apart from `basic_auth` and `digest_auth`, is there any plans to support `OAuth` and `OAuth2` ?

Thanks

**Comment by pared** [comment]
Hi @raman-rajarathinam! 
Currently, we do not have an open issue for that. Could you open one? If we get enough demand for that feature, we definitely would consider that. Also, we are open to contributors pull requests, and can help with development, if one decides to follow this path.
